packages/dev/s2-docs/pages/s2/Migrating.mdx

```diff
@@ -275,6 +273,7 @@ No updates needed.
 - Remove `validationState="valid"` (it is no longer supported in Spectrum 2)
 - Update `Item` to be a `PickerItem`
 - Change `isLoading` to `loadingState` and provide the appropriate loading state.
+- Update `defaultSelectedKey` and `selectedKey` to be `defaultValue` and `value` respectively. See the [Picker](Picker.html#value) documentation for more details.
```

Comment:
Want to update the storybook version of this migrating doc too?

Reply:
Should be handled by https://github.com/adobe/react-spectrum/pull/8946

Reply:
ah it didn't change the placeholder stuff though looks like